.github/workflows/flutter.yaml

```diff
@@ -13,4 +13,4 @@ jobs:
       - run: flutter pub get
       - run: flutter analyze
       - name: Run tests
-        run: flutter test
\ No newline at end of file
+        run: flutter test -r expanded
```

Comment:
This makes CI log formatting a bit nicer -- we see all the test cases that were executed.